src/Symfony/Component/PropertyAccess/StringUtil.php

```diff
@@ -203,7 +203,7 @@ public static function singularify($plural)
         }
 
         // Convert teeth to tooth, feet to foot
-        if (false !== ($pos = strpos($plural, 'ee')) && strlen($plural) > 3 && 'feedback' !== $plural) {
+        if (false !== ($pos = strpos($plural, 'ee')) && strlen($plural) > 3 && !preg_match('{feedback|Feedback|career|Career}', $plural)) {
```

Comment:
It should be at least 4 times because I would like to keep this as much narrowed as possible in order to avoid unwanted matches, such "FeedBacktrace" by instance.
BTW, I haven't any benchmark on this.